examples/cpp/put-get-remove/main.cpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <iostream>
+
+// Include the Geode libraries.
+#include <geode/CacheFactory.hpp>
+#include <geode/PoolManager.hpp>
+
+// Use the "geode" namespace.
+using namespace apache::geode::client;
+
+
+int main(int argc, char** argv) {
+
+  auto cacheFactory = CacheFactory();
+  cacheFactory.set("log-level", "none");
+  auto cache = cacheFactory.create();
+
+  auto poolFactory = cache.getPoolManager().createFactory();
+  poolFactory->addLocator("localhost", 10334);
+  auto pool = poolFactory->create("pool");
+
+  auto regionFactory = cache.createRegionFactory(PROXY);
+
+  auto region = regionFactory.setPoolName("pool").create("orders");
```

Comment:
Why are we making up a region name like this when the data below isn't even remotely an "order"? This should simply patch the project name to avoid collision with other examples. In fact, I would suggest "examples-put-get-remove".

Reply:
This is a good point. How about something like user_id and name.  Region can simply be region<string, string>

Reply:
sure, usually an order id and quantity are the base of an order, but ok. I have no strong preference. user_id and name it is.